### ISSUE TITLE
Add Read Usage Statistic on Cardinality Explorer page

### ISSUE THREAD

**Issue by tenmozes** [VictoriaMetrics/VictoriaMetrics#6145]
### Is your feature request related to a problem? Please describe

It would be nice to have statistics for metric names about their presence in query, query_range and export read responses.

If the count is zero or close to zero, the user can make a decision about removing such metrics

### Describe the solution you'd like

vmselect can track the metric names during requests. During retrieving data for Cardinality Explorer populates it with the number of calls per each top metric

**Comment by hagen1778** [comment]
Integration recommendations.

The statistics for metirc name usage can be requested via `http://<vmselect>/select/0/prometheus/api/v1/status/metric_names_stats` HTTP call. The response will look like the following:
```
{
  "status": "success",
  "statsCollectedSince": 1741370802,
  "statsCollectedRecordsTotal": 6643,
  "trackerMemoryMaxSizeBytes": 51539604,
  "trackerCurrentMemoryUsageBytes": 657198,
  "records": [
    {
      "metricName": "aggregator_discovery_aggregation_count_total",
      "queryRequestsCount": 0,
      "lastQueryRequestTimestamp": 0
    },
    {
      "metricName": "aggregator_unavailable_apiservice",
      "queryRequestsCount": 0,
      "lastQueryRequestTimestamp": 0
    },
...
```
Here:
* `status` determines the success of the fetch operation
* `statsCollectedSince` determines the timestamp when stats collection started
* `statsCollectedRecordsTotal` is the total number of entries in the cache
* `trackerMemoryMaxSizeBytes` is the max memory that can be used by tracker
* `trackerCurrentMemoryUsageBytes` is the current memory usage
* `records` is a list of returned metric objects:
* * `metricName` is a metric name
* * `queryRequestsCount` the number of times this metric was queried since `statsCollectedSince`
* * `lastQueryRequestTimestamp` is the timestamp when this metric name was queried last time.

Using this API I suggest the following changes to Cardinality explorer:
* On the cardinality explorer load page issue an additional request to `/select/<tenantID>/prometheus/api/v1/status/metric_names_stats`, where <tenantID> should be inherited from vmui settings
* the response size could take hundreds of KBs, so it is better to cache it for significant amount of time (15min or more)
* there is no need to refresh the cache on caridnality explorer date changes, because these stats are global and don't depend on date
* if request fails: do nothing and don't raise errors, as this API can be disabled.
* add a new panel next to `Total series` named `Metric names tracker`
<img width="365" alt="Image" src="https://github.com/user-attachments/assets/214c0104-bc1f-442d-a4d8-38f81c4dc0cd" />

* `Metric names tracker` should show the number of names in stats tracker  `statsCollectedRecordsTotal` and contain info button.
* info button should on-hover show a popup with info: `statsCollectedSince`, cache utilization `trackerCurrentMemoryUsageBytes / trackerMemoryMaxSizeBytes * 100` in %, link to https://docs.victoriametrics.com/victoriametrics/#track-ingested-metrics-usage
* if request to stats failed, the new panel should show 0 and popup should explain that tracker is likely disabled and point to the docs https://docs.victoriametrics.com/victoriametrics/#track-ingested-metrics-usage
* Table `Metric names with the highest number of series` should get a new column `Used in queries`. The column should be visible only if request to the stats API was successful. The column should have a description `Shows the number of times the metric name was used in read querie Metric names tracker.`
* Every metric name in column `Used in queries` should show human readable (shortened) `queryRequestsCount` and humar readable `lastQueryRequestTimestamp` in form `Last time queried at #v`.
* This column should support frontend sorting

----------------------

@f41gh7 there are complications to the implementations described above:
1. There is no way to request list of metric names from `Metric names tracker`, as `match_pattern` param doesn't support that. Ideally, vmui should reflect only metric names that are displayed on the page. But because of how  `match_pattern` is implemented, it would have to load all at once.

**Comment by f41gh7** [comment]
> There is no way to request list of metric names from Metric names tracker, as match_pattern param doesn't support that. Ideally, vmui should reflect only metric names that are displayed on the page. But because of how match_pattern is implemented, it would have to load all at once.


 I think, we could extend current API with metric_names param - comma-separated list of metric names. It should improve it's usability.

**Comment by f41gh7** [comment]
I also think of the reverse feature - return firstN metric names with `queryRequestsCount=0` and  add button - `show cardinality` for each metric name.

**Comment by hagen1778** [comment]
> I also think of the reverse feature - return firstN metric names with queryRequestsCount=0 and add button - show cardinality for each metric name.

Yes, this deserves a separate web page. But I was thinking to add it later, once we get integration with cardinality explorer.

**Comment by f41gh7** [comment]
The feature was included into [v1.116.0](https://docs.victoriametrics.com/changelog/#v11160)
